src/data/downloadMetadata.ts

```diff
@@ -2035,9 +2035,9 @@ export const dataPages: DownloadMetadata = {
     layerId: 0,
   },
   'Utah Roads': {
-    itemId: '478fbef62481427f95a3510a4707b24a',
+    itemId: '99lidPhWCzftIe9K',
```

Comment:
**issue**: itemid should never change

Reply:
another odd one, but the original itemid doesn't appear to work. maybe i'm missing something?

this works:
https://services1.arcgis.com/99lidPhWCzftIe9K/arcgis/rest/services/UtahRoads/FeatureServer/0

This does not:
https://services1.arcgis.com/478fbef62481427f95a3510a4707b24a/arcgis/rest/services/UtahRoads/FeatureServer/0

Reply:
Itemids are for arcgis hub. You are showing arcgis server urls. Totally separate.

Reply:
gotcha.  i fixed it. thanks